CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 - Add `find_earliest_finalized`, `find_instance_creation`,
   `find_first_finalized_block_no_later_than` methods to the `v2` client.
 - Bump MSRV to 1.60
+- Add deprecation notices to `ModuleRef` and `Parameter`. Use `ModuleReference`
+  and `OwnedParameter`, respectively, instead.
+  - There are no breaking changes to the API of either type, but `OwnedParameter` has some
```

Comment:
Isn't there a breaking change in that `Parameter` used to have `AsRef<Vec<u8>>` and now there is `AsRef<&[u8]>` only?

Reply:
Ah yes, I forgot about that one. Thanks!